docs/agent_container_manual_deploy.md

```diff
@@ -0,0 +1,76 @@
+## Instructions for starting an agent in a container on Linux
+These instructions assume that the agent container you wish to deploy already exists and is in a container image repository somewhere.
+
+### Prerequisites
+Docker needs to be installed on the host device. For instructions on installing Docker on Linux see [here](https://docs.docker.com/engine/install/).
+
+Need to have the SSL certificate from the management hub.
```

Comment:
Clarify this statement, the SSL cert is actually optional if the mgmt hub isnt using SSL.

Reply:
Done